external-api/src/helpers/object.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if an object is empty.
```

Comment:
Looking at this again especially as we have a test suite for this can use https://www.geeksforgeeks.org/lodash-_-isempty-method/ and https://www.geeksforgeeks.org/lodash-_-isobject-method/ instead of hand rolling this function? Should be a fairly quick refactor to just use those methods whre we call this function instead of this function

Reply:
Completely agree, will look into in a separate PR across all the micro-services, `_.isObject()` does sound like a good alternative.